mods/stairs/init.lua

```diff
@@ -468,13 +468,13 @@ end
 -- Nodes will be called stairs:{stair,slab}_<subname>
 
 function stairs.register_stair_and_slab(subname, recipeitem, groups, images,
-		desc_stair, desc_slab, sounds, worldaligntex)
+		desc_stair, desc_slab, sounds, worldaligntex, full_desc_stair_outer, full_desc_stair_inner)
```

Comment:
Line too long (80 column limit)
I also suggest calling these `desc_stair_outer`,`desc_stair_inner` because that's shorter.